test/sagemaker_tests/huggingface_pytorch/training/integration/local/test_single_machine_training.py

```diff
@@ -26,8 +26,10 @@
 @pytest.mark.skip_trcomp_containers
 @pytest.mark.team("sagemaker-1p-algorithms")
 def test_distilbert_base(
-    docker_image, processor, instance_type, sagemaker_local_session, py_version
+    docker_image, processor, instance_type, sagemaker_local_session, py_version, framework_version
 ):
+    if "pytorch" in docker_image and Version(framework_version) < Version("2.2"):
+        pytest.skip("Skipping distilbert SM local tests for PT")
```

Comment:
nit: maybe move the 2.2 to a constant `DISTILBERT_SM_MIN_VERSION`and make this comment to skip versions below it